opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java

```diff
@@ -80,6 +80,25 @@ void activateSpan() {
     assertThat(io.opentelemetry.api.baggage.Baggage.current()).isSameAs(EMPTY_BAGGAGE);
   }
 
+  @Test
+  void activateNullSpan() {
```

Comment:
What about starting the test with an actual active `Span`, _and then_ you activate/test the null value?

For testing purposes it's the same, but I get the feeling that changing it that way could make the test more readable.